Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [qTox_enhanced/qTox](https://hosted.weblate.org/projects/qtox_enhanced/qtox_app/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/qtox_enhanced/qtox_app/horizontal-auto.svg)

jiaowobaba02 committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (703 of 703 strings)

Translation: qTox_enhanced/qTox
Translate-URL: https://hosted.weblate.org/projects/qtox_enhanced/qtox_app/zh_Hans/

jiaowobaba02 reviewed (commented):
I have reviewed it,although it's my translation

jiaowobaba02 committed:
Translated using Weblate (Chinese (Traditional))

Currently translated at 24.1% (170 of 703 strings)

Translation: qTox_enhanced/qTox
Translate-URL: https://hosted.weblate.org/projects/qtox_enhanced/qtox_app/zh_Hant/

jiaowobaba02 committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (703 of 703 strings)

Translation: qTox_enhanced/qTox
Translate-URL: https://hosted.weblate.org/projects/qtox_enhanced/qtox_app/zh_Hans/

nikolio committed:
Translated using Weblate (Greek)

Currently translated at 78.6% (553 of 703 strings)

Translation: qTox_enhanced/qTox
Translate-URL: https://hosted.weblate.org/projects/qtox_enhanced/qtox_app/el/
